<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Anaglyph</class>
 <widget class="QDialog" name="Anaglyph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1306</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Anaglyph</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1301</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>645</width>
         <height>457</height>
        </rect>
       </property>
       <widget class="QLabel" name="label_left">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>641</width>
          <height>451</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>644</width>
         <height>457</height>
        </rect>
       </property>
       <widget class="QLabel" name="label_right">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>641</width>
          <height>451</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>460</y>
     <width>781</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="ButtonLeft">
      <property name="text">
       <string>Load &amp;Left</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ButtonRight">
      <property name="text">
       <string>Load &amp;Right</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="ButtonGenerate">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>520</y>
     <width>321</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Generate Anaglyph</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonExit">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>520</y>
     <width>95</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>ButtonExit</sender>
   <signal>clicked()</signal>
   <receiver>Anaglyph</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>777</x>
     <y>533</y>
    </hint>
    <hint type="destinationlabel">
     <x>431</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
